### PR TITLE
Added support for Clinical Reports.

### PR DESCRIPTION
**Pull request by quentinleguennec**
## Status
READY

## Migrations
NO

## Description
Added support for Clinical Records.

## Steps to Test or Reproduce
Fetch this branch and run the example app.
There is now a new blue icon next to the existing permission icon. Tapping it will request permissions for Clinical Records.
There is also a new button to query Clinical Records.

## Impacted Areas in Application
It will now be possible to request permissions to read Clinical Records (as any other permission), and to query Clinical Records (as any other sample).

### CODE REVIEW

**Comment by kvs-coder** [comment]
Hi @quentinleguennec 

thank you very much for your PR and please sorry again for my late response, I recently merged your PR of the original HealthKitReporter and released a 3.1.0 version to CocoaPods

**Review by kvs-coder — APPROVED** [review]

**Comment by quentinleguennec** [comment]
Thanks @kvs-coder, feel free to merge this PR if you did not find any issue with it (I cannot merge it myself).

**Pull request merged**
